SAST-JP/c101007058.lua

```diff
@@ -0,0 +1,79 @@
+--Trickstar Live Stage
+--Logical Nonsense
```

Comment:
If this is based on the anime version, make sure you include the original credit here too (if it wasn't or it didn't have one, this is fine)

Reply:
Yeah, I forgot to factored in anime cards as a reference when I was scripting this card. So it was from scrap.